tests/cross_functional/system_test/test_nfs_export_in_out_cluster.py

```
@@ -0,0 +1,2001 @@
+        log.info("Step 3: Getting NFS export details for cloned PVC")
+
+        # Use helper function to get export details
+        cloned_export = self.get_nfs_export_details(cloned_pvc_name)
+        cloned_share_details = cloned_export["share_details"]
+
+        # Use helper function to mount NFS export
+        cloned_test_folder = f"{self.test_folder}-cloned"
+        self.mount_nfs_export(con, cloned_share_details, cloned_test_folder)
+
+        # Add cloned mount to cleanup
+        def cleanup_cloned_mount():
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `cleanup_cloned_mount`

Add return type annotation: `None`

(ANN202)